### DISCUSSION TITLE
Version 9.8.1.0

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#676] [Announcements]
<ul>
  <li>Changes
      <ul>
          <li>For custom dictionaries, adapt to the official Bing API</li>
          <li>Now the button to switch between translation services also appears in the <i>simple interface</i> when the site is translated</li>
      </ul>
  </li>
  <li>Fixed
      <ul>
          <li><b>Copy</b> button does not appear when translating selected text from inputs</li>
          <li>Translate button in context menu does not appear in local files</li>
          <li>Text files are not translated if <b>&lt;pre&gt;</b> tag translation is disabled</li>
      </ul>
  </li>
  <li>Update interface translations</li>
</ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.8.1.0'>Version 9.8.1.0</a>.</em>

**Comment by NayamAmarshe** [comment]
Any plans to publish on Chrome Web Store? :)

**Comment by msdnxh** [comment]
安卓端可以做成小图标吗？就是侧边栏收缩的那种，也可以做成顶部栏出现的那种吗？
